Makefile

```diff
@@ -42,10 +42,3 @@ endif
 rmcerts:
 	@rm -f apiserver-key.pem apiserver.csr apiserver.pem
 	@rm -f metrics-ca-config.json metrics-ca.crt metrics-ca.key
-
-.PHONY: deploy
```

Comment:
Please undo the makefile changes, they are not related to this PR.

Reply:
ok